web/styles/compose.css

```diff
@@ -342,8 +342,32 @@
         var(--color-message-content-container-border-over-limit);
 }
 
+.edit-content-container .message_edit_content.over-limit,
+.edit-content-container .message_edit_content.over-limit:focus {
```

Comment:
I missed this before, but CSS for message edit content is in message_row.css so might be good to move all the new CSS for it there. Atleast this CSS which is just for message edit UI. Maybe we can also move the other CSS which is same for both compose and message edit UI with small duplication.

Reply:
Hey @sahil839 , when I move this css to `message_row.css`, then the css is not applied to the respected area for some reason as below;

![image](https://github.com/user-attachments/assets/0dc67178-9483-4660-b96e-57e48a78c3ec)

Reply:
Why the CSS is required for `:focus` as well?

Reply:
:focus is required because when the focus is not on the textarea, then the css for it is not implemented. 
I tried removing :focus class, but I ran into the problem i mentioned.

Reply:
Also, after moving the css into `message_edit_content`, I had to write the classes as

    &.message_edit_content.over-limit,
    &.message_edit_content.over-limit:focus 
    &.message_edit_content.approaching-limit,
    &.message_edit_content.approaching-limit:focus 
    
    because writing them as 
    
    &.over-limit,
    &.over-limit:focus 
    &.approaching-limit,
    &.approaching-limit:focus 
    
    did not work for dark theme when we remove focus.

Reply:
Also, now I have removed the `:focus` pseudo class, it has no visible effect on the UI.